codegen/core/src/test/resources/test-files/codegen-annotation/generate-from-samples.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+# Re-generates sample proto file descriptors from the source
+# Requires https://github.com/lightbend/akkaserverless-java-sdk to be cloned locally, and its path provided
```

Comment:
Why is the Java SDK needed, couldn't it just invoke npm build for each of the samples and then pick up the updated proto desc insteads?

Reply:
This is wrong, I will remove it. Also from the Java SDK. 

At one point, we added the new annotations in the SDK and then I needed it. I forgot to remove this when I put the protos back in the framework.

Reply:
Done in e49069b